pkgs/top-level/all-packages.nix

```diff
@@ -11324,6 +11324,11 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  russ = callPackage ../applications/networking/feedreaders/russ {
+    inherit (xorg) libxcb;
```

Comment:
We removed most of these a while ago.

```suggestion
```

Reply:
Funny, because I just changed it to match the surrounding packages in `all-packages.nix`.